tests/e2e/cleantest_test.go

```diff
@@ -0,0 +1,145 @@
+package e2e
+
+import (
+	"math/rand"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/odo/tests/e2e/helper"
+)
+
+//  current directory and project (before eny test is run) so it can restored  after all testing is done
+var originalDir string
+var originalProject string
+
+// BeforeSuite is run before whole test suite
+// there can be only one BeforeSuite
+var _ = BeforeSuite(func() {
+	// seed pseudo-random generator
+	rand.Seed(time.Now().UTC().UnixNano())
+
+	// this needs to bedeclared separately
+	// if := is used in the next line it won't save originalDir to global variable
+	var err error
+	originalDir, err = os.Getwd()
+	Expect(err).NotTo(HaveOccurred())
+
+	originalProject = helper.OcCurrentProject()
+})
```

Comment:
If these will be ran for each and every test, would it make sense to try to put as much information as possible into the utility functions / helper files?